samples/net/sockets/mtls_http_get/src/ca_certificate.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __CA_CERTIFICATE_H__
+#define __CA_CERTIFICATE_H__
+
+#define CA_CERTIFICATE_TAG 1
+
+/* By default only certificates in DER format are supported. If you want to use
```

Comment:
This comment seems in conflict with the PEM format certificates these files point to.